README.md

```diff
@@ -52,3 +52,31 @@ test server binary, demo.c, and kflow.h.
 All built artifacts will be placed in `$CURDIR/out/$TARGET` where $CURDIR
 is the current working directory and $TARGET is `$GOOS-$GOARCH`, for
 example `linux-amd64`.
+
+## Building library for internal use cases
+
+Currently only supported when run from macOS.
+
+Internal use cases require a version of libkflow targeting a variety of
+architectures laid out in a specific directory structure. 
+
+First, checkout the desired version from git and ensure it is tagged correctly. 
+```shell
+git tag -a $VERSION
+```
+
+First, ensure that the musl cross compilers are installed. This can be done
+with the command:
+
+```shell
+brew install filosottile/musl-cross/musl-cross
+```
+
+Then, run the following command to build the libraries:
+
+```shell
+make internal-libs
```

Comment:
chore (non-blocking cause can't block): This won't work after checking out an older version, since that older version of the branch doesn't have this target. I think that's perfectly fine, just an FYI of something I hit

Reply:
Thanks for pointing that out.

Reply:
Resolving to allow merging. Be sure to tag primary branch once merged.